src/docs/arch-fast-truncate.dox

```diff
@@ -64,32 +64,22 @@ Apart from instantiated pages, whose state is \c WT_REF_MEM (see @ref
 ft_instantiated_pages below) the \c page_del field will be \c NULL.
 
 Because locking the ref is expensive, and visibility checks are expensive as well, we want
```

Comment:
I would rephrase this as "Locking the ref and visibility checks are expensive, we want to avoid doing either unnecessarily."